Stores/PageStore/Sources/PageStore/PageStore.swift

```diff
@@ -30,6 +30,7 @@ public final class PageStore: ObservableObject {
     // MARK: - Substores & Services
 
     let dataManager: PageDataManagerProtocol
+    private var model = QuestionsViewModel()
     
     // MARK: - Public properties
```

Comment:
Here you can add the `@Published public private(set) var loadingMore: Bool = false` to manage a loader of more data.